packages/content/test/notifications/ThirdPartyRelationshipAttributeDeletedByPeerNotificationItem.test.ts

```diff
@@ -0,0 +1,10 @@
+import { Serializable } from "@js-soft/ts-serval";
+import { ThirdPartyRelationshipAttributeDeletedByPeerNotificationItem } from "../../src";
+
+test("ThirdPartyRelationshipAttributeDeletedByPeerNotificationItem", () => {
```

Comment:
that test title is a bit misleading.

wtb 

1. make the test a describe
2. add two tests describing that the class ThirdPartyRelationshipAttributeDeletedByPeerNotificationItem can be created with the respecting `@type`